packages/harpguru-core/src/components/zoom-slide-vertical/hooks/use-styles/use-styles.ts

```diff
@@ -18,8 +18,8 @@ export const useStyles = (
   columnCount: number
 ): Styles => {
   const {
-    dynamicSizes: { legendWidth, zoomSlideWidth },
-  } = useSizes()
+    [0]: { legendWidth, zoomSlideWidth },
+  } = useGlobal('dynamicSizes')
```

Comment:
Similar point to https://github.com/js-jslog/harpguru/pull/137#discussion_r765155671

Reply:
Done